freshdesk/manifest.json

```diff
@@ -1,12 +1,16 @@
 {
-  "platform-version": "2.3",
```

Comment:
@Saif-Shines Please upload the platform v2.3 app is pushed as a separate branch with the name "platform-v2.3" for public view.
3.0 changes can be pushed to the main branch and developers can refer to both the version while building apps.